### ISSUE TITLE
HW 2 Feedback

### ISSUE THREAD

**Issue by alexchaomander** [selwyth/DAT_SF_12#4]
- Nice use of plots to help you determine the number of neighbors for KNN
- You can make use of a pivot table (http://pandas.pydata.org/pandas-docs/stable/reshaping.html) to be able to better and more compactly see the predicted vs. actual
- There are other ways to calculate the accuracy of your model. For instance most models will have a score function. i.e. "clf.score(X_test, y_test)" http://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html
- But what you wrote essentially does that albeit in a more manual way. It's good practice to write out the entire calculation so that you know the internals behind functions. 
- Check out the solutions note about scaling. Turns out that if you scale your variables before running kmeans, you can get much better predictive accuracy!
- And while the problem didn't specify it, I think using a few basic histograms to visualize the distribution of the data can go a long way in catching things like the differing scales and other peculiarities with which you can modify your model.
- All in all, good work!
